client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -445,55 +497,69 @@ const updateTheme = ( siteId, themeId ) =>
 		}
 	);
 
-const mapStateToProps = ( state, { siteId, plugins, themes } ) => {
+/**
+ * Start updating WordPress core on the specified site.
+ *
+ * @param {number} siteId  Site Id.
+ *
+ * @return {*} Stored data container for request.
+ */
+const updateCore = siteId =>
+	requestHttpData(
+		`core-update-${ siteId }`,
+		http( {
+			method: 'POST',
+			path: `/sites/${ siteId }/core/update`,
+			// No need to pass version: if it's missing, WP will be updated to latest core version.
+		} ),
+		{
+			fromApi: () => ( { version } ) => {
+				return [ [ version, true ] ];
+			},
+			freshness: -Infinity,
```

Comment:
@dmsnell Could something like that be just assumed if the user is doing a `POST` request?

Reply:
that's a reasonable assumption. I'm not sure we have enough warrant to conclude that entirely yet.

in any case for a `POST` I think it would make sense to have `freshness: Infinity` since we don't want to inadvertently resubmit these requests.

`-Infinity` means "resubmit the request if the last call took place earlier than the infinite future"